src/components/Main/Main.js

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
 import './Main.css';
+import '../Animal/Animal.css';
+
+import background from './background.png';
+import Animal from '../Animal/Animal.js';
+import { animals } from '../../data.js';
 
 export default function Main() {
-  return <main></main>;
+  return (
+    <div className="main" style={{ backgroundImage: `url(${background})` }}>
+      <main />
+      {animals.map((item) => (<Animal key={item.name} {...item} />))}
+    </div>
```

Comment:
Great work Brenden! Within our <Animal /> component, we typically want to pass props as name={item.name} says={item.says} type={item.type} left={item.left} top={item.top}. Then in Animal.js, we can pass props as the argument and utilize dot notation to render elements, such as name === props.name. We will learn destructuring soon, I just want to be sure you fully understand passing down props prior to destructuring.